docs/How-to-Guides/book-on-tictactrip/tutorial.md

```diff
@@ -361,7 +362,8 @@ Once you have validated that the booking was successful by **[getting the order]
 You must pass the id of the order and the file name you want to give to the file as a parameters in the url.
```

Comment:
I'd probably specify that the default name provided if the argument is lacking is `ticket.pdf`

Reply:
It's wrote in the full request description.

Still need to add it here ?

Reply:
I would, but it's your call.

I'll approve the PR.